apps/obs-web/src/views/DebrieferAssessment.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+    <div>
+        <p style="font-weight: bold; font-size: 20px; letter-spacing: .5px;">Fishing Effort:</p>
+        <div style="margin-left: 20px">
+            # vessels: {{ uniqueVessels }}<br>
+            # Trips: {{ debriefer.trips.length }}<br>
+            # Hauls: {{ totalHauls }}
+        </div>
+        <div v-for="section of sections" :key="sections.indexOf(section)">
+            <p style="font-weight: bold; font-size: 20px; letter-spacing: .5px; margin-top: 1em;">{{section}}</p>
+            <div v-for="question of questions" :key="questions.indexOf(question)">
+                <div v-if="question.section === section" class="row" style="margin-left: 20px">
+                    <span class="col5" style="position: relative; top: 10px; width: 400px">{{ question.question }}&nbsp; &nbsp;</span>
+                    <div class="col6">
```

Comment:
When you click to input text it would be nice if the dropdown showed immediately rather than having to click on the dropdown arrow. And it would be nice if you could type and it would filter the options similar to how the observer selection works (although backspace isn't working there and I need to fix that)

Reply:
I've updated so clicking in the field now brings up the options (and clicking the field again without making a choice makes them go away).

I'm not sure typing to filter options is possible the way I've implemented (QInput with QMenu) - when the menu is displaying, the input does not have focus - so keystrokes are not added to the model.  Given a max of 6 options, maybe we don't really need filtering?

Reply:
Okay yeah that looks good now, it seems pretty easy to select an options so I agree text filtering isn't needed. Just curious how come you used QInput and QMenu rather than just using QSelect? Seems like that would be easier to use. 

Also like how you added the cards to separate each section that looks good.